src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/python/schema_validation.py

```diff
@@ -333,7 +333,10 @@
 TARGET_SCHEMA = {
     Optional("path"): Or(str, int, TARGET_LIST_SCHEMA),
     Optional("tags"): {
-        And(str, Regex(r"\A.{1,128}\Z")): And(str, Regex(r"\A.{0,256}\Z"))
+        And(str, Regex(r"\A.{1,128}\Z")): Or(
+            And(str, Regex(r"\A.{0,256}\Z")),
+            And(list)
```

Comment:
I think we should also validate the items in this list.
The [Schema Example](https://github.com/keleshev/schema#example) shows this would be possible I think:

```suggestion
            [And(str, Regex(r"\A.{0,256}\Z"))]
```